src/App.js

```diff
@@ -45,52 +87,47 @@ class Image extends React.Component {
                     return response.json();
                 })
                 .then((arrayOfData) => {
-                    this.setState(prevState => ({
-                        images: prevState.images.concat(arrayOfData),
-                        isFetching: false
-                    }));
+                    this.setState((prevState) => {
+                        const buffArray = prevState.images.concat(arrayOfData)
+                        return {
+                            count: buffArray.length,
+                            images: buffArray,
+                            isFetching: false
+                        }
+                    });
                 });
         }
     }
 
-
     render() {
+        const divStyle = {
+            position: 'sticky',
+            top: '20px',
+            marginLeft: '400px',
+            width: '200px',
+        };
+
+        const handleCountChange = (countToDelete) => {
+            this.setState({countToDelete: countToDelete});
+        }
+
         const cats = this.state.images.map((data) => {
             return (
                 <div key={data.id}>
                     <img src={data.url} height={IMAGE_HEIGHT} alt="could not download"/>
                 </div>)
         });
-        return (
-            <div>
-                {cats}
-            </div>
-        );
-    }
-}
-
-function Counter(props){
-    const handleClick = () => {
 
+        const countToDelete = this.state.countToDelete;
```

Comment:
Minor. It's usually more readable to use object destructuring:
`const { countToDelete, count } = this.state;`